app/src/main/java/chat/rocket/android/chatroom/adapter/ChatRoomAdapter.kt

```diff
@@ -18,7 +20,8 @@ class ChatRoomAdapter(
     private val roomName: String? = null,
     private val presenter: ChatRoomPresenter? = null,
     private val enableActions: Boolean = true,
-    private val reactionListener: EmojiReactionListener? = null
+    private val reactionListener: EmojiReactionListener? = null,
+    private val activity: FragmentActivity? = null
```

Comment:
What about declaring the `context: Context? = null` instead of  `activity: FragmentActivity?` here?
Could you change it?

Reply:
Done :+1: